custom/icds_reports/ucr/data_sources/home_visit_forms.json

```diff
@@ -24,11 +24,11 @@
   "mirrored_engine_ids": [
     {
       "server_environment": "icds",
-      "engine_ids": ["icds-ucr-citus"]
```

Comment:
also similar issues here in these smaller non dashboard data sources.

I think that this is fine because this PR keeps the canonical database the same. If you want to move these data sources to be backed off of citus as the primary, then you'll need to re-migrate these tables or rebuild their data sources